### DISCUSSION TITLE
How to change colours

### DISCUSSION THREAD

**Discussion by kametsun** [LazyVim/LazyVim#3980] [Q&A]
I installed lazyvim using the installation instructions in the documentation.
I would like to know how to change the colour of the red and yellow areas in this screenshot.

environment:
OS: mac OS
nvim: 0.10.0

![image](https://github.com/LazyVim/LazyVim/assets/127584315/0ac0bbb2-fb85-4605-827e-1c4d8df291f9)

**Comment by folke** [comment]
check the docs on suggested terminals
